### PR TITLE
Hybrid buttons

### PR DESCRIPTION
**Pull request by elliottcrush**
#### What does this PR do? (please provide any background)

- [x] Upgrades the hybrid brand to use the new hunter colours.

- [x] Add in secondary colour support

 - [x] Map to, but retain hx-btn-ultimate for the time being

#### How can this be tested?
`cd pattern-library && npm link ../brands`

#### What gif best describes how you feel about this work?

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [x] :+1:
- [x] UXUI Team :+1:
- [x] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked all the work locally on pattern library.

---

### CODE REVIEW

File: brands/hybrid/src/less/colours.less [hunk 1]
```diff
@@ -4,16 +4,15 @@
 // --------------------------------------------------
 
 // Core brand colours
-@logo-purple:            rgb(102,0,153);
-@logo-yellow:            rgb(255,221,0);
+@color-hx-purple :      rgb(84,46,145);       // #542e91
```
Comment (jodiedoubleday) [inline brands/hybrid/src/less/colours.less]: picky but no need for spaces before semi-colon.. nothing else has it.

**Comment by jodiedoubleday** [comment]
One small comment, nothing serious :+1:

File: brands/hybrid/src/less/colours.less (hunk 1 above)
Comment (elliottcrush) [inline brands/hybrid/src/less/colours.less]: Agreed.

**Comment by h0rhay** [comment]
changes look good 👍

**Comment by h0rhay** [comment]
Nice all good from me 👍

**Pull request merged**